### ISSUE TITLE
line numbers misaligned on canvas cards

### ISSUE THREAD

**Issue by cptnjangles** [kepano/obsidian-minimal#541]
### Is the bug present when using the default Obsidian theme?

No

### Is the bug present when snippets and plugins are disabled?

Yes

### Minimal theme version

6.2.2

### Describe the bug

 line numbers overlapping the first character while using canvas
![Screenshot_2023-02-13_at_1 34 03_pm](https://user-images.githubusercontent.com/61850101/218890791-d1c53d52-69a7-4d01-ab9d-ab66225f8b8a.png)

### Debug info

some text

**Comment by chodorowicz** [comment]
I've added this snippet to custom CSS snippets to move the content a bit to the right.

```
.node-insert-event.mod-inside-iframe .cm-editor .cm-content {
    margin-left: 20px;
}
```

**Comment by cptnjangles** [comment]
Hi, appreciate the update. Issue seems to be fixed on iOS, tested on iPad Pro 12.9 and iPhone 13 Pro Max. However not fixed on MacOS. Any ideas? Cheers
